spec/factories/tasks.rb

```diff
@@ -1,5 +1,8 @@
 FactoryBot.define do
   factory :task do
-    
+    sequence(:title){ |n| "やる事#{n}" }
+    content { "本文" } 
+    status  { "todo" }
```

Comment:
enumの値はシンボルで定義しておきましょう。
```suggestion
    status  { :todo }
```

Reply:
2ae78f

Reply:
a02bca8c91943c1afd447abd00d0009eb7efb401
enumの値をシンボルに変更致しました。